tests/drivers/dma/chan_blen_transfer/boards/s32z2xxdc2_s32z270_rtu1.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/memory-attr/memory-attr-arm.h>
+
+/ {
+	soc {
+		sram1: memory@35780000 {
```

Comment:
You don't need to re-define the whole node here, just use something like this (but out of devicetree root `/{ }` )
```
&sram1 {
     reg = <0x31780000 DT_SIZE_K(960)>;
}
```
Do this for all places. Then please mention in the commit message that the non-cacheable memory used for DMA tests is split from the system SRAM

Reply:
Thanks, I updated.

Reply:
Where is this update `Then please mention in the commit message that the non-cacheable memory used for DMA tests is split from the system SRAM`? put something like that in commit msg or in code pls

Reply:
I updated the commit msg also.